### PR TITLE
completed prep-css-display exercise

### PR DESCRIPTION
**Pull request by gabriellerramos**
Screen Shot 2022-09-21 at 4 40 53 PM

### CODE REVIEW

File: prep-css-display/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    Contact Us
+  </li></ul>
+  <style>
+ ul{
+  border: 4px solid purple;
+}
+li{
+  border:4px solid magenta;
+  display: inline;
+}
```
Comment (thebearingedge) [inline prep-css-display/index.html]: Hi Gabrielle, for this exercise, we want to write all of our CSS code inside one <style> element in the <head> of the document.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by gabriellerramos** [comment]
Screen Shot 2022-09-27 at 6 04 41 PM

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Gabrielle, we'll want to make sure that our HTML document has all of the necessary parts to be valid. The style can go inside the `<head>` below the `<title>`

```html
<!doctype html>
<html>
<head>
  <title>CSS Display Properties</title>
  <style>
    /* CSS code goes here */
  </style>
</head>
<body>
  <!-- visible HTML content goes here -->
</body>
</html
```

**Comment by gabriellerramos** [comment]
Screen Shot 2022-10-03 at 1 04 19 AM

File: prep-css-display/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<head>
+  <title>CSS Display Properties</title>
+  <style>
+</head>
```
Comment (thebearingedge) [inline prep-css-display/index.html]: The closing tag for the head element should go after the closing tag for the style element.

File: prep-css-display/index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      <div> I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
```
Comment (thebearingedge) [inline prep-css-display/index.html:6]: The div and ul elements should all go in the body of the document, you can see an example here

https://github.com/gabriellerramos/prep-0922-code-solutions/pull/8#pullrequestreview-1122856949

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by gabriellerramos** [comment]
Screen Shot 2022-10-05 at 9 40 06 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**